src/plain_data.rs

```diff
@@ -18,7 +18,7 @@
 use rustc_serialize::{Decoder, Encodable, Encoder};
 use xor_name::XorName;
 
-/// PlainData
+/// Plain data with a name and a value.
```

Comment:
Yes I did, but it is fine Andreas, much better shape now. Thx for these PR's they have helped a lot.